pkg/pb/gc.proto

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package gc;
+
+
+import "gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+service GarbageCollection {
+    rpc Retain(RetainRequest) returns (RetainResponse);
+}
+
+message Filter {
+    uint32 seed = 1;
+    uint32 k = 2;
+    uint32 bits_per_element = 3;
+    bytes table = 4;
```

Comment:
How is the storage node supposed to create a `bloomfilter.Filter` struct from this protobuf?

`bloomfilter.NewFilter` from #2113 is taking `expectedElements int, p float64` params only.